internal/rpc/client_test.go

```diff
@@ -518,12 +518,12 @@ func handleWS(f func(*websocket.Conn) error) http.Handler {
 		defer c.Close()
 		err = f(c)
 		var cm []byte
+		closeError, isCloseError := err.(*websocket.CloseError)
```

Comment:
maybe instead of this you can user `errors.Is`?

Reply:
I need to cast the type in anycase to return the format the close message in `websocket.FormatCloseMessage(closeError.Code, closeError.Text)`